### PR TITLE
Add fix for ServerError after moving to .NET5

### PR DESCRIPTION
**Pull request by Mpdreamz**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,6 +25,9 @@ jobs:
         git fetch --prune --unshallow --tags
         echo exit code $?
         git tag --list
+    - uses: actions/setup-dotnet@v1
+      with:
+          dotnet-version: '3.1.201'
```
Comment (stevejgordon) [inline .github/workflows/ci.yml:30]: Is this still required?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -34,6 +37,8 @@ jobs:
         
     - run: ./build.sh build -s true
       name: Build
+    - run: ./build.sh test -s true
+      name: Test
```
Comment (stevejgordon) [inline .github/workflows/ci.yml:41]: Good idea!

**Review by stevejgordon — APPROVED** [review]
Not sure why we still need the 3.1.x SDK on build in this repo.

Actions are complaining about missing XML comments.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Mpdreamz) [reply .github/workflows/ci.yml]: Nope good catch!

**Review by stevejgordon — APPROVED** [review]
LGTM

**Pull request merged**